src/types.ts

```diff
@@ -76,3 +76,31 @@ export type OnAnimationEnd = {
 export type OnFinish = {
 	isTrusted: boolean;
 };
+
+export type ActiveInput = {
+	start: string[],
+	move: string[],
+	end: string[],
+};
+export type InputEventType = PointerEvent | MouseEvent | TouchEvent;
+
+export type PanEvent = {
+	srcEvent: InputEventType;
+	angle: number;
+	center: {
+		x: number;
```

Comment:
Please change the indent type to space(with size 2) in the next update.
`.editorconfig` and VS code's `"Convert Indentation to Spaces"` can help.

Reply:
I think linting changes should be made in main v3 branch, Since we have multiple branches for updating v3 and linting in one branch can cause several merge conflicts.